etc/docs/migration-v4.md

```diff
@@ -6,16 +6,193 @@
 
 - [Overview](#overview) 🔭
   - [What has changed](#what-has-changed)
-  - [What has improved](#new-additions)
-  - [What was fixed](#what-was-fixed)
+  - [What we added](#new-additions)
 
 # Overview 🔭
 
 ## What has changed?
 
+### 🎨 `Colors`
+
+<img width="983" alt="Screenshot 2020-03-27 at 18 30 26" src="https://user-images.githubusercontent.com/5938217/77783386-0fefde80-7059-11ea-979a-bf3c11f28d27.png">
+
+Colours have changed again. Here's a mapping table hopefully easy your pain through the migration:
+
+| **v3**                     |        **v4**         |
+| -------------------------- | :-------------------: |
+| `colors.brand.primary`     |    `colors.brand`     |
+| `colors.brand.secondary`   | `colors.actionPlain`  |
+| `colors.neutral.white`     |    `colors.white`     |
+| `colors.neutral.nearWhite` | `colors.greyLightest` |
+| `colors.neutral.light`     | `colors.greyLighter`  |
+| `colors.neutral.medium`    |  `colors.greyLight`   |
+| `colors.neutral.nearDark`  |   `colors.greyDark`   |
+| `colors.neutral.dark`      | `colors.greyDarkest`  |
+
+You can [check the documentation](https://zopauk.github.io/react-components/#/Content?id=spacing) for all the updated colour spectrum.
+
+### ☄️ `Buttons`
+
+One of the biggest changes on the streamline, our `<Button />` component has been massively simplified and got few extra features.
+
+You only now have these 4 states:
+
+- **primary**
+- **secondary**
+- **disabled**
+- **text**
+- **loading**
+
+[Check te docs](https://zopauk.github.io/react-components/#/Components/Atoms/Button) to see the new available states and ask **#design** or **#ui-kit-web** if one of your use-cases is not supported.
+
+🚨 **Notice:**
+
+You should make use of the new state:
+
+```jsx
+<Button loading />
+```
+
+and move away from any custom solution you were using for this.
+
+### 🅰 `Typography`
+
+#### Import
+
+The library doesn't `import` anymore **Open Sans** for you, the `<Fonts />` component has been removed.
+
+Add a manual link to your root HTML instead:
+
+```html
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700;800&display=swap" rel="stylesheet" />
+```
+
+#### Components
+
+```jsx
+import { Heading, Text, Link } from '@zopauk/react-components';
+```
+
+These components retain their API, however `<Heading />`, `<Text />` and `<Link />` have lost their `mb` prop, use [the new atomic classes](#new-additions) instead.
+Since `<Heading />`'s `mb` prop was set to `true` by default, you should add `className="mb-6"` to maintain the existing styling.
+The default for `<Text />` and `<Link />` was `false` so you can just find and replace `mb` prop with `className="mb-6"`.
+
+#### Font sizes
```

Comment:
Should we say something about _lead_ here?

Reply:
`lead` was there before so I'm not sure if it's worth it

Reply:
👍